tests/pkg/ops/tikv.go

```diff
@@ -53,41 +57,54 @@ func (ops *TiKVOps) TruncateSSTFile(opts TruncateOptions) error {
 		})
 	}
 
-	stdout, stderr, err := exec("find", "/var/lib/tikv/db", "-name", "*.sst", "-o", "-name", "*.save")
-	if err != nil {
-		glog.Errorf("list sst files: stderr=%s err=%s", stderr, err.Error())
-		return errors.Annotate(err, "list sst files")
-	}
+	retryCount := 0
+	for ; retryCount < retryLimit; retryCount++ {
+		if retryCount > 0 {
+			time.Sleep(10 * time.Second)
+		}
+		stdout, stderr, err := exec("find", "/var/lib/tikv/db", "-name", "*.sst", "-o", "-name", "*.save")
+		if err != nil {
+			glog.Warningf("list sst files: stderr=%s err=%s", stderr, err.Error())
+			continue
+		}
 
-	sstCandidates := make(map[string]bool)
+		sstCandidates := make(map[string]bool)
```

Comment:
delete the blanks

Reply:
the one more indent because it's in retry loop now.